CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff

```diff
@@ -6,23 +6,23 @@
     </header>
     <body>
       <trans-unit id="CFBundleDisplayName" xml:space="preserve">
-        <source>Callsheet</source>
+        <source>Callsheet</source><target>Callsheet</target>
```

Comment:
BTW, most conflicts get resolve if you replace `</source><target>`

with
```
</source>
<target state="translated">
```

Note the line break.

Reply:
(I guess `state="translated"` is optional, but the "missing" line break confuses gits diff.

Reply:
Thanks for the info, I'll see what I can do